backend/src/main/java/me/performancereservation/api/RefundController.java

```diff
@@ -0,0 +1,72 @@
+package me.performancereservation.api;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import me.performancereservation.domain.refund.RefundService;
+import me.performancereservation.domain.refund.dto.RefundDetailResponse;
+import me.performancereservation.domain.refund.dto.RefundRequest;
+import me.performancereservation.domain.refund.enums.RefundStatus;
+import me.performancereservation.global.exception.AppException;
+import me.performancereservation.global.exception.ErrorCode;
+import me.performancereservation.global.exception.ErrorType;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Slf4j
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/api/v1/refunds")
+public class RefundController {
+
+    private final RefundService refundService;
+
+    /*--- USER 요청에 대응 ---*/
+    // 본인 id와 일치하는 모든 환불내역 리스트 반환
+    @GetMapping("/user/{userId}")
+    public ResponseEntity<List<RefundDetailResponse>> getAllRefundDetailsWithUserId(@PathVariable Long userId) {
+        return ResponseEntity.status(HttpStatus.OK)
+                .body(refundService.findAllRefundsDetailByUserId(userId));
+    }
+
+    // 특정 예약id 환불요청 -> 환불내역 생성
+    @PostMapping("/user/new-refund")
```

Comment:
```suggestion
    @PostMapping
```

이것도 마찬가지로 /api/v1/refunds 에 환불 요청을 하는게 어떨까요?
RESTful 관점에서 보면 'refunds라는 리소스에 새로운 객체를 생성한다'로 해석이 가능하거든요

참고로 api endpoint에는 '행위'는 최대한 안추가하는게 좋습니다

Reply:
좋습니다 수정 완료했습니다!